### DISCUSSION TITLE
Editing FAQ page

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1535] [Q&A]
1. I have tried to edit Ukrainian version for the FAQ page, but it was unsuccesful.
Could you, please, take a look - what was wrong here?
2. How can I add a hyperlink to the text?

---
title: Frequently Asked Questions (FAQ)
permalink: /uk/faq/
layout: page
language: uk
---

## Зміст
- [Питання 1](#Question-1)

## Питання 1?
Відповідь на питання 1.

[Back to top](#top)

**Comment by brockfanning** [comment]
@ukraine-sdg The test failure has a clue as to what's going on: https://github.com/ukraine-sdg/site/runs/4782735549?check_suite_focus=true

It looks like the "html proofer" tests are failing because there is an anchor link pointing to "#Question-1", but that anchor does not exist. You may need to remove that anchor link in order to get the tests to pass, so that you can merge the pull request.

Regarding hyperlinks, that syntax you used is correct, if you would like to use Markdown. Here is more information on adding links in Markdown syntax: https://www.markdownguide.org/basic-syntax/#links

You are also welcome to use HTML links, if you would like (for example, `<a href="https://un.org">UN website</a>`.

**Comment by LucyGwilliamAdmin** [comment]
@ukraine-sdg has Brock's response resolved this for you?
